freetypefont.mod/freetypefont.bmx

```diff
@@ -169,12 +169,21 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 			Local data:Byte[1024 * 90]
 			Local dataSize:Int
 
+			'backup old stream position to set it back to it afterwards
+			'Pos() returns -1 for non-seekable-streams
+			Local oldStreamPos:int = stream.Pos()

```

Comment:
As far as I am aware, there's no other code that does this with streams in any of the modules.